extensions/amp-story/1.0/live-story-manager.js

```diff
@@ -64,37 +69,28 @@ export class LiveStoryManager {
       'amp-story must contain id to use the live story functionality'
     );
 
+    Services.extensionsFor(this.ampdoc_.win).installExtensionForDoc(
+      this.ampdoc_,
+      'amp-live-list'
+    );
```

Comment:
I think we should one of two things here:
1. Enforce that the extension is present at validation level
2. If we decide to lazy load it, we should do so at a better time, when all the first pages assets are loaded. There's a custom even that's dispatched for that

My vote goes to (1), since we really want publishers to understand that using a live-story is expensive: it loads and parses (!!!) the full HTML document every 15s for every single user viewing it, and loads more code.

Reply:
I think @newmuis prefers if we dynamically add it ourselves 
https://github.com/ampproject/amphtml/pull/23041#discussion_r297878247